src/absil/il.fsi

```diff
@@ -1638,7 +1644,7 @@ val splitNamespace: string -> string list
 
 val splitNamespaceToArray: string -> string[]
 
-/// The splitILTypeName utility helps you split a string representing
+/// The <c>splitILTypeName</c> utility helps you split a string representing
```

Comment:
FWIW I don't mind the use of ``<c>`` on true code identifiers.  I just found it a bit odd on ``true`` :)